YesterdayApiTest/ExceptionHandlingMiddlewareTest.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Xunit;
+using YesterdayApi.Utilities.Exceptions;
+
+namespace YesterdayApiTest
+{
+    public class ExceptionHandlingMiddlewareTest
+    {
+        [Fact]
+        public async Task CustomExceptionIsRaised_HandleItToCustomErrorResponseAndCorrectHttpStatus()
```

Comment:
warning CA1707: Remove the underscores from member name

Please turn of this warning